gradle.properties

```diff
@@ -1,6 +1,6 @@
 GROUP=com.squareup.misk
 
-VERSION_NAME=0.5.0-SNAPSHOT
+VERSION_NAME=0.7.0-SNAPSHOT
```

Comment:
0.6.0 or 0.7.0?

Reply:
I screwed up 0.5.0 so that version number is permanent burned!